frontend/src/components/nav/Nav.js

```diff
@@ -24,17 +25,20 @@ const logout = () => {
       <div className="nav-container">
           {(loggedIn ?
           <div className="nav-box">
-            <h1>Title</h1>
-            <NavButton to="/posts" value="Posts"/>
+            <h1><i>AceBook</i></h1>
+
+            <button className="post-button" onClick={Feed}>Posts</button>
```

Comment:
do we need to import the feed in here? can we not reroute using the url address?